invenio_theme/shared.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 Graz University of Technology.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio standard theme."""
+
+from flask_menu.menu import MenuRoot
+
+menu = MenuRoot("", None)
```

Comment:
```suggestion
from flask_menu import Menu

menu = MenuRoot("", None)
menu_ext = Menu(root=menu)
```

major: see https://github.com/inveniosoftware/flask-menu/pull/85/files#r1250872290

Reply:
Why i need `menu_ext` here?

Reply:
For consistency with the Flask pattern we're going for tbh... I was even thinking that we could "proxy" the `menu.submenu(...)` call using a new `Menu.submenu()` method to avoid having to export a `MenuNode` object instead of the extension... But this can be easily done also later, once we know that it all works :tm:

Reply:
i personally don't like the `Menu.submenu()` approach. It feels like a singleton and i used them to often in the past. in my opinion they produce more problems that they solve.

the `menu_ext` will not be used somewhere so it is only to initialize the `Menu` on the parsing the python file level? 

is it really necessary to initialize `Menu` directly? i think i didn't do that somewhere and the tests are all green also on other packages that are using the PR.